catalogue/webapp/pages/work.js

```diff
@@ -188,7 +189,11 @@ const WorkPage = ({work}: Props) => {
       <PageDescription title='Search our images' extraClasses='page-description--hidden' />
       <InfoBanner text={`Coming from Wellcome Images? All freely available images have now been moved to the Wellcome Collection website. Here we're working to improve data quality, search relevance and tools to help you use these images more easily`} cookieName='WC_wellcomeImagesRedirect' />
 
-      <WorkMedia2 id={work.id} iiifUrl={iiifInfoUrl} title={work.title} />
+      <WorkMedia2
```

Comment:
seems strange to have this coupled into `WorkMedia`, but we can change that later.